syntax = "proto2";

package mesop.components.datepicker;

// Next ID: 14
message DatePickerType {
  optional string value = 1;
  optional string placeholder = 2;
  optional bool disabled = 3;
  optional bool required = 4;
  optional bool readonly = 5;
  optional bool hide_required_marker = 6;
  optional string color = 7;
  optional string float_label = 8;
  optional string appearance = 9;
  optional string subscript_sizing = 10;
  optional string hint_label = 11;
  optional string label = 12;
  optional string on_change_handler_id = 13;
}
